CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 - ([#445](https://github.com/demos-europe/demosplan-ui/pull/445)) **Breaking:** Remove DpDashboardTaskCard component ([@hwiem](https://github.com/hwiem))
 
 ### Changed
+- ([#472](https://github.com/demos-europe/demosplan-ui/pull/472)) Migrate DpCard Storybook to v7 ([@ahmad-demos](https://github.com/@ahmad-demos))
```

Comment:
The docs were not there before, so i'd sugest to add an "Added" section and change the description to "Add DpCard documentation to Storybook" or so.

Reply:
https://github.com/demos-europe/demosplan-ui/pull/472/commits/786a65c1ec231d5be714711793bb8f33a91bf684